main.go

```diff
@@ -57,6 +59,10 @@ func run(coverprofile string, args []string, covermode, cpu, parallel, timeout s
 		usage()
 		return nil
 	}
+	if race && covermode != "" && covermode != "atomic" {
+		return fmt.Errorf("cannot use race flag and covermode=%s. See more detail on golang/go#12118.", covermode)
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
error strings should not be capitalized or end with punctuation or a newline